src/components/results/sensitivity-analysis/non-evacuated-energy/non-evacuated-energy-result-tab.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { FunctionComponent, SyntheticEvent, useState } from 'react';
+import { NonEvacuatedEnergyTabProps } from './non-evacuated-energy-result.type';
+import { NonEvacuatedEnergyResult } from './non-evacuated-energy-result';
+import { useNodeData } from '../../../study-container';
+import { fetchNonEvacuatedEnergyResult } from '../../../../services/study/non-evacuated-energy';
+import { Box } from '@mui/system';
+import { useOpenLoaderShortWait } from '../../../dialogs/commons/handle-loader';
+import { LinearProgress, Tab, Tabs } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { RunningStatus } from '../../../utils/running-status';
+import { ComputationReportViewer } from '../../common/computation-report-viewer';
+import { REPORT_TYPES } from '../../../utils/report-type';
+import { useSelector } from 'react-redux';
+import { ReduxState } from '../../../../redux/reducer.type';
+import { ComputingType } from '../../../computing-status/computing-type';
+import { RESULTS_LOADING_DELAY } from '../../../network/constants';
+
+const styles = {
+    container: {
+        display: 'flex',
+        position: 'relative',
+    },
+    tabs: {
+        position: 'relative',
+        top: 0,
+        left: 0,
+    },
+    loader: {
+        height: '4px',
+    },
+    resultContainer: {
+        flexGrow: 1,
+    },
+};
+
+export const NON_EVACUATED_ENERGY_RESULT_INVALIDATIONS = [
+    'nonEvacuatedEnergyResult',
+];
+
+export const NonEvacuatedEnergyResultTab: FunctionComponent<
+    NonEvacuatedEnergyTabProps
+> = ({ studyUuid, nodeUuid }) => {
```

Comment:
```suggestion
type ResultTabInputs = {
    studyUuid: UUID;
    nodeUuid?: UUID;
};

export const NonEvacuatedEnergyResultTab = ({
    studyUuid,
    nodeUuid,
}: ResultTabInputs) => {
```
It would be more idiomatic as well